tests/kernels/test_activation.py

```diff
@@ -7,22 +7,26 @@
 NUM_TOKENS = [7, 83, 2048]  # Arbitrary values for testing
 D = [512, 4096, 5120, 13824]  # Arbitrary values for testing
 SEEDS = [0]
+DEVICES = [i for i in range(torch.cuda.device_count())]
```

Comment:
Can we just test the kernels for device 0 and 1? This will help shorten the test time.